abnf/odata-abnf-construction-rules.txt

```diff
@@ -298,27 +298,33 @@ expandItem        = "$value"
                   / expandPath
                   / optionallyQualifiedEntityTypeName "/" expandPath
 expandPath        = ( STAR [ ref / OPEN levels CLOSE ]
-                    / ( navigationProperty / entityAnnotationInQuery ) [ "/" optionallyQualifiedEntityTypeName ]
-                      [ ref   [ OPEN expandRefOption   *( SEMI expandRefOption   ) CLOSE ]
-                      / count [ OPEN expandCountOption *( SEMI expandCountOption ) CLOSE ]
-                      /         OPEN expandOption      *( SEMI expandOption      ) CLOSE
+                    / entityNavigationProperty [ "/" optionallyQualifiedEntityTypeName ]
+                      [ ref 
+                        / OPEN entityOption *( SEMI entityOption ) CLOSE

```

Comment:
```suggestion
                      / OPEN entityOption *( SEMI entityOption ) CLOSE

```
We align `/` with the surrounding parentheses or brackets

Reply:
fixed.